index.js

```diff
@@ -1,8 +1,37 @@
+const dark=document.querySelector(".dark");
 var slide = document.querySelector("#slider");
 var btn1 = document.querySelector("#btn1");
 var btn2 = document.querySelector("#btn2");
 var btn3 = document.querySelector("#btn3");
 
```

Comment:
Using multiple document.querySelector can cause performance issues in case of a big application.
The better way would be to use document.querySelector for a parent element and then use that element to query instead of 'document'.

Reply:
ok, I will do that. Thank you for the review